vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2825,19 +2861,13 @@ private void setTopic() {
      * @param aTopicValue the new topic value
      */
     private void setTopic(String aTopicValue) {
-        // in search mode, the topic is not displayed
-        if (!TextUtils.isEmpty(mEventId)) {
-            mActionBarCustomTopic.setVisibility(View.GONE);
-        } else {
-            // update the action bar topic anyway
-            mActionBarCustomTopic.setText(aTopicValue);
+        mActionBarTopic.setText(aTopicValue);
```

Comment:
move this line in the else condition :
```suggestion
        // topic is only displayed if its content is not empty
        if (TextUtils.isEmpty(aTopicValue)) {
            mActionBarTopic.setVisibility(View.GONE);
        } else {
            mActionBarTopic.setText(aTopicValue);
            mActionBarTopic.setVisibility(View.VISIBLE);
        }
```